src/libsync/checksums.cpp

```diff
@@ -120,6 +132,12 @@ QByteArray contentChecksumType()
     return type;
 }
 
+QList<QByteArray> collisionSafeHashes()
+{
+    static QList<QByteArray> list = { "SHA1" };
```

Comment:
SHA1 may not be safe against attacks, but collisions are still overwhelmingly unlikely. Even MD5 should be okay.